libfwupd/fwupd-security-attr.h

```diff
@@ -88,6 +88,8 @@ typedef enum {
  * @FWUPD_SECURITY_ATTR_RESULT_NOT_FOUND:		NOt found
  * @FWUPD_SECURITY_ATTR_RESULT_SUPPORTED:		Supported
  * @FWUPD_SECURITY_ATTR_RESULT_NOT_SUPPORTED:		Not supported
+ * @FWUPD_SECURITY_ATTR_RESULT_SECURE:			Secure
```

Comment:
I think this is a risky word to use.  It's only going to be secure in the sense that the current known vulnerabilities from when the daemon was last compiled/updated.
I would err on something like "No known vulnerabilities", or "Vulnerabilities present".

Reply:
Ya, I wasn't sure about that to. Got any good alternatives to `SECURE`? Perhaps we just VALID for no issues, and NOT_VALID when known issues are present?

Reply:
Well I was thinking vulnerabilities/no vulnerabilities is better since it can be overloaded into other things that might be measured for vulnerabilities later.

Reply:
The other thing I pondered was making the HSI from a two dimensional thing (e.g. firmware platform security * runtime issues) into a three dimensional thing (firmware platform security * runtime issues * known security issues) but I think that's making it too complicated. The user can't fix the firmware if there is a known CSME vulnerability so it's not runtime, and we've said that the system can be downgraded HSI version after launch as we add new tests, so I guess it makes sense the platform HSI can be downgraded if the vendor is not "quick enough" to build a new ME during the CVE embargo period.

Reply:
>but I think that's making it too complicated

Yeah 3d is too complex I agree.

>The user can't fix the firmware if there is a known CSME vulnerability so it's not runtime, and we've said that the system can be downgraded HSI version after launch as we add new tests, 

Which is part of why I think it will be important to document exactly which CVEs or vulnerabilities are being checked for/measured in the spec.  If a new one is added, it goes in the spec and it's obvious in the code the changes needed for it.

>so I guess it makes sense the platform HSI can be downgraded if the vendor is not "quick enough" to build a new ME during the CVE embargo period.

Sometimes it's not just this, but if it causes a genuine regression that needs to be fixed.

Reply:
Thinking about this a bit more, I think we could just print "OK" for the "no vulnerability found" message, and "Not valid" for the case where we think there might be a CSME issue.

Reply:
That seems fine